Sources/TensorFlow/Layers/Convolutional.swift

```diff
@@ -485,3 +485,121 @@ public extension TransposedConv2D {
             padding: padding)
     }
 }
+
+/// A 2-D depthwise convolution layer for spatial depthwise convolution over images.
+///
+/// This layer creates seperable convolution filters that are convolved with the layer input to produce a
+/// tensor of outputs.
+@frozen
+public struct DepthwiseConv2D<Scalar: TensorFlowFloatingPoint>: Layer {
+    /// The 4-D convolution kernel for the depthwise convolution.
+    public var depthwise_kernel: Tensor<Scalar>
```

Comment:
Please use camel case for all variable names.

Reply:
@rxwei yes I will, but as far as the names itself are concerned, did I get them right?

Reply:
In this particular case, since the kernel property is defined inside type called `DepthwiseConv2D`, I don't see any reason the property should mention the word "depthwise". Therefore, `kernel` is fine.